make/devices.cmake

```diff
@@ -358,7 +358,7 @@ function(DEFINE_DEVICE)
         ${symbiflow-arch-defs_SOURCE_DIR}/common/wire.eblif
         ${DEVICE_MERGED_FILE} ${DEVICE_MERGED_FILE_TARGET}
         ${QUIET_CMD} ${QUIET_CMD_TARGET}
-        ${VPR} ${VPR_TARGET}
```

Comment:
You do actually want the executable in the depends list.  For example, if you recompile yosys or vpr, this target needs to be rebuilt.  So this was correct originally.

Reply:
That makes sense now, I had not understood exactly why the dependency on the executable was there (if the target was handling the build/rebuild). Should the `VPR` like variables simply point at the full path?

Reply:
Yes, the executable variables (e.g. ${VPR}, ${YOSYS}) should be full paths generally speaking.  There is an exception, which is that if the executable is built by the CMake enviroment, the variable should be the target name from add_executable.